dotCMS/src/main/java/com/dotmarketing/portlets/containers/action/EditContainerAction.java

```diff
@@ -1,7 +1,11 @@
 package com.dotmarketing.portlets.containers.action;
 
+import com.dotmarketing.beans.MultiTree;
+import com.dotmarketing.business.FactoryLocator;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotmarketing.business.FactoryLocator'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)